resources.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<!--
+	Hyperspace by HTML5 UP
+	html5up.net | @ajlkn
+	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
+-->
+<html>
+  <head>
+    <title>CODEVID-19 - Resources</title>
+    <meta charset="utf-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, user-scalable=no"
+    />
+    <link rel="stylesheet" href="assets/css/main.css" />
+    <noscript><link rel="stylesheet" href="assets/css/noscript.css"/></noscript>
+  </head>
+  <body class="is-preload">
+    <!-- Header -->
+    <header id="header">
+      <a href="index.html" class="title">Hyperspace</a>
+      <!-- Need to turn this into a shared element we can use on all pages. For now sync by hand. YUCK! -->
+      <nav>
+        <ul>
+          <li><a href="index.html">Home</a></li>
+        </ul>
+      </nav>
+    </header>
+
+    <!-- Wrapper -->
+    <div id="wrapper">
+      <!-- Main -->
+      <section id="main" class="wrapper">
+        <div class="inner">
+          <h1 class="major">Resources & Offers</h1>
+          <ul class="actions stacked">
+            <li><a href="https://github.com/devedmonton/codevid19.com/edit/master/resources.html" class="button primary fit">EDIT THIS PAGE ON GITHUB TO ADD NEW RESOURCES AND OFFERS</a></li>
```

Comment:
I think the button styles make it all-caps without needing to type it in all-caps, I could be wrong though.